tools/cldr-apps/js/src/esm/cldrTable.mjs

```diff
@@ -707,7 +735,7 @@ function updateRowCodeCell(tr, theRow, cell) {
   ) {
     cldrSurvey.appendExtraAttributes(cell, theRow);
   }
-  if (CLDR_TABLE_DEBUG) {
+  if (CLDR_TABLE_DEBUG_ZOOM) {
```

Comment:
If enabled, the first column gets "zoom{JSON}" links -- what are they for? They don't seem to accomplish anything with the current back end, and I don't find "r_rxt" or "rxt" in the current Java code. If this code is obsolete, let's delete it

Reply:
they were removed some time back.